master/getting-started/kubernetes/installation/hardway/the-calico-data-store.md

```diff
@@ -0,0 +1,109 @@
+---
+title: The Calico data store
+canonical_url: 'https://docs.projectcalico.org/v3.8/getting-started/kubernetes/installation/hardway/the-calico-data-store'
+---
+
+{{site.prodname}} stores the data about the operational and configuration state of your cluster in a central data store. If the data store is unavailable
+your {{site.prodname}} network continues operating, but cannot be updated (no new pods can be networked, no policy changes can be applied, etc.).
+
+{{site.prodname}} has two data store drivers you can choose from
+
+- **etcd** - for direct connection to an etcd cluster 
+- **Kubernetes** - for connection to a Kubernetes API Server
+
+## Using Kubernetes as the data store
+
+This guide uses the Kubernetes data store driver. The advantages of this driver when using {{site.prodname}} on Kubernetes are
+
+- Doesn't require an extra data store, so is simpler to manage
+- You can use Kubernetes RBAC to control access to {{site.prodname}} resources
+- You can use Kubernetes audit logging to generate audit logs of changes to {{site.prodname}} resources
+
+For completeness, the advantages of the etcd driver are
+
+- Allows you to run {{site.prodname}} on non-Kubernetes platforms (e.g. OpenStack)
+- Allows separation of concerns between Kubernetes and {{site.prodname}} resources, for example allowing you to scale the data stores independently
+
+## Custom Resources
+
+When using the Kubernetes data store driver, most {{site.prodname}} resources are stored as 
+[Kubernetes custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+
+A few {{site.prodname}} resources are not stored as custom resources and instead are backed by corresponding native Kubernetes resources. For example, [Workload Endpoints](/{{page.version}}/reference/resources/workloadendpoint) are Kubernetes Pods.
+
+In order to use Kubernetes as the {{site.prodname}} data store, we need to define the custom resources {{site.prodname}} uses.
+
+Download and examine the list of {{site.prodname}} Custom Resource Definitions, and open it in a file editor.
+
+```
+wget {{site.url}}/{{page.version}}/manifests/crds.yaml
```

Comment:
I don't have a preference. They are the "stock" CRDs, so I thought it was fine to have in the main directory.

Reply:
Given the design for now is to not use our templates for this, I suggest we don't create a new file for this and instead inline the contents like the rest of the manifests in this PR.

If that's not something you think we should do then I think I'd prefer a `hardway` subdirectory so that it's clear what purpose this file has. (e.g., Users which deploy in etcd mode shouldn't use the files, but they are in the same directory as the etcd manifest, which might be confusing for some). 

I think in the past is has been helpful to encode this extra bit of context into the URL.